test/e2e/density.go

```diff
@@ -47,11 +47,27 @@ import (
 )
 
 const (
-	MinSaturationThreshold     = 2 * time.Minute
-	MinPodsPerSecondThroughput = 8
-	DensityPollInterval        = 10 * time.Second
+	MinSaturationThreshold        = 2 * time.Minute
+	MinPodsPerSecondThroughput    = 8
+	DensityPollInterval           = 10 * time.Second
+	deploymentResource            = "deployment"
+	replicaSetResource            = "replicaSet"
+	replicationControllerResource = "replicationController"
 )
 
+func runResource(resource string, config testutils.RCConfig) error {
```

Comment:
Or maybe even ```config interface{}``` and then:
switch config.(type):
case RCConfig:
  ...
case RSConfig:
  ...
...
}
```